### PR TITLE
Various changes

### PR DESCRIPTION
**Pull request by sibayanjasper**

### CODE REVIEW

File: app/album/__init__.py [hunk 1]
```diff
@@ -85,6 +92,31 @@ def get_all_albums(_params):
...
+        database.query(db.music_db, 'INSERT INTO album_user_ratings(`album_id`, \
+            `user_id`, `rating`) VALUES(:album_id, :user_id, :rating)', _params)
+
+    data = database.get(db.music_db, 'SELECT rating FROM album_user_ratings WHERE album_id = :album_id', _params)
+
+    rating = 0
+    for row in data:
+        rating += int(row['rating'])
+
+    _params['album_rating'] = str(round(rating / float(len(data)), 1))
```
Comment (pprmint) [inline app/album/__init__.py]: See proper grouping of spaces.
Look at python documentation on anytv github

File: app/artist/__init__.py [hunk 2]
```diff
@@ -31,11 +31,18 @@ def add_artist(_params):
 
         async_upload_image(bucket.new_key(path), file_string)
 
-    data = database.query(db.music_db, 'INSERT INTO artists VALUES(:artist_id, :name, :picture, :date_created, NULL)', _params)
+    data = database.query(db.music_db, 'INSERT INTO artists VALUES(:artist_id, \
```
Comment (pprmint) [inline app/artist/__init__.py]: Use proper new line as indicated in the documentation.
You may use Python Auto format for this.

File: app/artist/__init__.py [hunk 3]
```diff
@@ -78,6 +86,32 @@ def get_artist_tracks(_params):
...
+        database.query(db.music_db, 'INSERT INTO artist_user_ratings(`artist_id`, \
+            `user_id`, `rating`) VALUES(:artist_id, :user_id, :rating)', _params)
+
+    data = database.get(db.music_db, 'SELECT rating FROM artist_user_ratings WHERE artist_id = :artist_id', _params)
+
+    rating = 0
+    for row in data:
+        rating += int(row['rating'])
+
+    _params['artist_rating'] = str(round(rating / float(len(data)), 1))
```
Comment (pprmint) [inline app/artist/__init__.py]: Use proper space grouping.
Use Auto Format

File: app/category/__init__.py [hunk 4]
```diff
@@ -11,6 +11,21 @@
...
 
     @staticmethod
+    def count_all_genre_tracks(_params):
+        data = database.get(db.music_db, 'SELECT COUNT(*) AS total_count FROM tracks WHERE uploaded = 1 AND \
+            genre LIKE CONCAT(CONCAT("%", (SELECT name FROM genres WHERE genre_id = :genre_id)), "%")', _params)
+
+        return data[0]['total_count']
+
+
+    @staticmethod
```
Comment (pprmint) [inline app/category/__init__.py]: I thought we are not using static methods?

Comment (sibayanjasper) [inline app/category/__init__.py]: Done. Python 3 can use class declarations without parentheses i.e. class <classname>:

File: app/artist/__init__.py (hunk 3 above)
Comment (sibayanjasper) [inline app/artist/__init__.py]: Done. Already used python pep8 autoformat.

File: app/artist/__init__.py (hunk 2 above)
Comment (sibayanjasper) [inline app/artist/__init__.py]: Done. Already used python pep8 autoformat.

**Pull request merged**